package.json

```diff
@@ -25,10 +25,14 @@
     "url": "https://github.com/IgniteUI/material-icons-extended/issues"
   },
   "homepage": "https://github.com/IgniteUI/material-icons-extended#readme",
+  "ng-update": {
+    "migrations": "./migrations/migration.json"
+  },
   "dependencies": {
     "@types/node": "^14.0.25"
```

Comment:
Completely unrelated, but are you sure types/node isn't a dev dep? All I see in the source are some interfaces/utils and exports; nothing node-specific.